### ISSUE TITLE
RESQUE_TERM_TIMEOUT env var is begin treated as a float

### ISSUE THREAD

**Issue by werkshy** [nevans/resque-pool#162]
We have `RESQUE_TERM_TIMEOUT=9` in our environment and we get this crash at exit.

```
Error
----------------
NoMethodError: undefined method `round' for "9999999999":String

Location
----------------
vendor/bundle/ruby/2.4.0/gems/resque-1.27.4/lib/resque/worker.rb:553 - wait_for_child_exit
```

The issue is that the env var isn't being converted to a float, so we are doing `"9"*10` which gives `"999999999"`.

**Comment by pjambet** [comment]
Fixed by #165 and #163

**Comment by nevans** [comment]
Sorry for being MIA for so long. #165 was merged a few months ago and v0.7.0 has been released just now. Let me know how it works for you, thanks!
